src/app/zap-templates/zcl/data-model/chip/boolean-sensor-configuration-cluster.xml

```diff
@@ -0,0 +1,71 @@
+<?xml version="1.0"?>
+<!--
+Copyright (c) 2023 Project CHIP Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<configurator>
+  <domain name="CHIP"/>
+
+  <bitmap name="Feature" type="bitmap32">
+    <cluster code="0x0080"/>
+    <field name="Visual" mask="0x1"/>
+    <field name="Audible" mask="0x2"/>
+    <field name="AlarmSupress" mask="0x4"/>
+    <field name="SensitivityLevel" mask="0x8"/>
+  </bitmap>
+
+  <bitmap name="AlarmModeBitmap" type="bitmap8">
+    <cluster code="0x0080"/>
+    <field name="Visual" mask="0x1"/>
+    <field name="Audible" mask="0x2"/>
+  </bitmap>
+
+  <enum name="SensitivityEnum" type="enum8">
+    <cluster code="0x0080"/>
+    <item name="High" value="0x0"/>
+    <item name="Standard" value="0x1"/>
+    <item name="Low" value="0x2"/>
+  </enum>
+
+  <cluster>
+    <name>Boolean Sensor Configuration</name>
+    <domain>Measurement &amp; Sensing</domain>
```

Comment:
Please mark all of these as provisional. 

@andreilitvin Can you help with syntax?

Reply:
do `<cluster apiMaturity="provisional">` for the cluster tag (see https://github.com/project-chip/connectedhomeip/blob/master/src/app/zap-templates/zcl/data-model/chip/scene.xml#L36 as an example).

We cannot (yet?) do tags on anything else as I have to add things one by one and zap reviews take a long time (e.g. attribute was next and it is standing for 1 week in https://github.com/project-chip/zap/pull/1169). This is good for me to know though ... after attributes it seems like I may want to tackle bitmaps next.

Reply:
done